pkgs/by-name/si/simple64/package.nix

```
@@ -84,13 +87,32 @@ stdenv.mkDerivation (finalAttrs: {
     mkdir -p $out/share/simple64 $out/bin
     cp -r simple64/* $out/share/simple64
 
+    mkdir -p $out/share/icons/hicolor/scalable/apps
+    install -Dm644 ./simple64-gui/icons/simple64.svg $out/share/icons/hicolor/scalable/apps/
```

nit: there is a way to do this in one line:

```suggestion
    install -Dm644 ./simple64-gui/icons/simple64.svg -t $out/share/icons/hicolor/scalable/apps/
```
or
```suggestion
    install -Dm644 ./simple64-gui/icons/simple64.svg $out/share/icons/hicolor/scalable/apps/simple64.svg
```

(the latter is mostly useful if the file copied needs to have a different name after copying)